frontend/src/pages/LearningCourse/FormDialog/FormDialog.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
Modal window should be as common component with an opportunity to change content and buttons